src/Illuminate/View/Compilers/ComponentTagCompiler.php

```diff
@@ -165,15 +176,15 @@ protected function componentString(string $component, array $attributes)
         if (! class_exists($class)) {
             $parameters = [
                 'view' => "'$class'",
-                'data' => '['.$this->attributesToString($data->all()).']',
+                'data' => '['.$this->attributesToString($data->all(), $escapeBound = false).']',
```

Comment:
This is currently not covered by any test (Changing to `$escapeBound = true` doesn't cause any tests to fail).

Reply:
Here is an attempt to cover this in `testClasslessComponents()`: https://github.com/perifer/framework/commit/46176c631faff707a5545ffb48199c8fb349690e

Reply:
Thanks